go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/docker/distribution v2.7.1+incompatible // indirect
 	github.com/docker/docker v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible // translates to v19.03.12
 	github.com/docker/go-connections v0.4.0
+	github.com/go-git/go-git/v5 v5.2.0
```

Comment:
Not a blocker per se, but unsure why we need a fullblown git implemention as a dependency? 🤔
Were there no other alternatives?